README.md

```diff
@@ -114,31 +114,43 @@ You can generate a PDF or an HTML copy of this guide using
     [1, 2, 3].length
     ```
 
-* Indent `when` as deep as `case`. I know that many would disagree
-  with this one, but it's the style established in both the "The Ruby
-  Programming Language" and "Programming Ruby".
-
-    ```Ruby
-    case
-    when song.name == 'Misty'
-      puts 'Not again!'
-    when song.duration > 120
-      puts 'Too long!'
-    when Time.now.hour > 21
-      puts "It's too late"
-    else
-      song.play
-    end
-
-    kind = case year
-           when 1850..1889 then 'Blues'
-           when 1890..1909 then 'Ragtime'
-           when 1910..1929 then 'New Orleans Jazz'
-           when 1930..1939 then 'Swing'
-           when 1940..1950 then 'Bebop'
-           else 'Jazz'
-           end
-    ```
+* Indent `when` consistently in the manner all other `XXX` ... `end` syntactic constructs
+  are indented.  Endeavor to use one-line alternatives and `then` as often as possible.
+  (Since methods are  *supposed*  to be short, we don't want multiline alternatives anyway.)
+
+  ```Ruby
+  # bad - unnecessary long; hard to scan conditions because of interleaved alternatives
+  case token.kind
+  when :star_op
+      stack.pop * stack.pop
+  when :slash_op
+      stack.pop / stack.pop
+  when :plus_op
+      stack.pop + stack.pop
+  when :minus_op
+      stack.pop - stack.pop
+  when :int_literal
+      token.value
+  end
+
+  # bad - lack of indentation hides control structure
+  case token
+  when :star_op then stack.pop * stack.pop
+  when :slash_op then stack.pop / stack.pop
+  when :plus_op then stack.pop + stack.pop
+  when :minus_op then stack.pop - stack.pop
+  when :int_literal then token.value
+  end
+
+  # good - concise; alternatives indented; conditions can be scanned vertically
+  case token
+      when :star_op then stack.pop * stack.pop
+      when :slash_op then stack.pop / stack.pop
+      when :plus_op then stack.pop + stack.pop
+      when :minus_op then stack.pop - stack.pop
+      when :int_literal then token.value
```

Comment:
I agree with this.  It looks like we removed the most-basic example of cases with multi-line actions.  

Also, there is no demonstration to confirm how to format when multiple conditions should result int he same action:

```Ruby
case token
   when :star_op
     stack.pop * stack.pop
   when :slash_op
     stack.pop / stack.pop
   when :minus_op
     also_calculate_that
     stack.pop - stack.pop
   when :plus_op,
        :plus_plus_op
     stack.pop + stack.pop
   when :int_literal 
     token.value
   end
```

which reads better than

```Ruby
# bad

case token
   when :star_op
     stack.pop * stack.pop
   when :plus_op, :plus_plus_op
     stack.pop + stack.pop
   when :int_literal, :str_literal, :str_interpolated 
     token.value
   end
```

Where the 'bad' example also has the issue of cause the entire when line to diff when one of the conditions is changed or updated